# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

BanditActionGoal action_goal
BanditActionResult action_result
BanditActionFeedback action_feedback
